about.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Web Development Course</title>
+
+    <!-- Link Lato font from Google fonts -->
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:400,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+
+    <!-- Link file for the cocogoose font -->
+    <link rel="stylesheet" href="fonts/cocogoose/Cocogoose.css" />
+
+    <!-- Global style file -->
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- Desktop style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (min-width: 768px)"
+      href="css/desktop.css"
+    />
+    <!-- Mobile style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (max-width: 768px)"
+      href="css/mobile.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="toggle-icon"></div>
+      <div class="menu">
+        <nav class="secondary-menu desktop-padding">
+          <ul>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-facebook.svg"
+                  alt="Facebook small icon"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-twitter.svg"
+                  alt="Twitter small icon"
+                />
+              </a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </nav>
+        <nav class="primary-menu desktop-padding">
+          <a href="/"><img src="images/logo.svg" alt="Logo" /></a>
+          <div class="primary-menu-content">
+            <ul>
+              <li><a class="menu-active" href="about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Register</a></li>
+              <li><a href="#">News</a></li>

```

Comment:
- I think it is important for you to add a link to the index page just as @Munsa1 has suggested. This will help you navigate back to the home page on your website mobile screen🤗